geonode/settings.py

```diff
@@ -898,14 +914,8 @@
     DEFAULT_FILE_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
     MEDIA_URL = "https://%s/%s/" % (AWS_S3_BUCKET_DOMAIN, MEDIAFILES_LOCATION)
 
-import djcelery
-djcelery.setup_loader()
 
-# Load more settings from a file called local_settings.py if it exists
```

Comment:
`local_settings.py` is the bad idea, I have had to chase my own tail in production several times because of `local_settings.py`.[1][2]

[1] http://nedbatchelder.com/blog/201112/duplicitous_django_settings.html
[2] https://zeth.net/archive/2011/12/31/django-and-the-localsettings-anti-pattern/